src/style.css

```diff
@@ -21,6 +21,13 @@
   border: 1px solid @borders;
 }
 
+button.pill.small {
+  font-size: 0.83333em;
+  border-radius: 99px;
+  margin: 0;
+  padding: 1px 12px;
```

Comment:
These are the styles from Builder's language tags: https://gitlab.gnome.org/GNOME/gnome-builder/-/blob/main/src/libide/gui/style.css?ref_type=heads#L33

It uses Libadwaita's `pill` class but makes it considerably smaller so it can act as a small tag.